common/logger_config.go

```diff
@@ -95,10 +95,10 @@ func ReadLoggerCLIConfig(ctx *cli.Context, flagPrefix string) (*LoggerConfig, er
 
 func NewLogger(cfg LoggerConfig) (logging.Logger, error) {
 	if cfg.Format == JSONLogFormat {
-		return logging.NewSlogJsonLogger(cfg.OutputWriter, &cfg.HandlerOpts), nil
```

Comment:
This constructor has been [deprecated](https://github.com/Layr-Labs/eigensdk-go/pull/218)